src/compas_rhino/artists/pointartist.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import compas_rhino
+from compas_rhino.artists import _PrimitiveArtist
+
+
+__all__ = ['PointArtist']
+
+
+def list_like(target, value, fillvalue=None):
```

Comment:
This one feels like an intruder from another PR. ;)

Reply:
yes i know i needed a temp fix, but will replace it as soon as available :)